src/openzaak/components/catalogi/models/eigenschap.py

```diff
@@ -29,15 +29,17 @@ class EigenschapSpecificatie(models.Model):
     ‘Referentie naar eigenschap’.
     """
 
-    groep = models.CharField(  # waardenverzameling Letters, cijfers en liggende streepjes
-        _("groep"),
-        max_length=32,
-        blank=True,
-        validators=[validate_letters_numbers_underscores_spaces],
-        help_text=_(
-            "Benaming van het object of groepattribuut waarvan de EIGENSCHAP een "
-            "inhoudelijk gegeven specificeert."
-        ),
+    groep = (
+        models.CharField(  # waardenverzameling Letters, cijfers en liggende streepjes
+            _("groep"),
+            max_length=32,
+            blank=True,
+            validators=[validate_letters_numbers_underscores_spaces],
+            help_text=_(
+                "Benaming van het object of groepattribuut waarvan de EIGENSCHAP een "
+                "inhoudelijk gegeven specificeert."
+            ),
+        )
```

Comment:
We could move the comment to above the field declaration to avoid this reformatting

Reply:
Done